src/Cli.Tests/ConfigureOptionsTests.cs

```diff
@@ -109,6 +109,130 @@ public void TestAddDepthLimitForGraphQL()
             Assert.AreEqual(maxDepthLimit, config.Runtime.GraphQL.DepthLimit);
         }
 
+        /// <summary>
+        /// Tests that running "dab configure --runtime.graphql" on a config with variuos values results
```

Comment:
Please update the comments such that it specifies what the test is doing and what are the expected results. For example:

```
This test validates that `dab configure --runtime.graphql` is correctly updating the config file with provided value of graphql flag.
```